### PR TITLE
added path in stderr

### PR DESCRIPTION
**Pull request by rajibmitra**
**What this PR does / why we need it**:
This PR adds the path in stderr message.
**Which issue(s) this PR fixes**
- Fixes #
N/A
**Special notes for your reviewer**:
I thought of adding the path in stderr message.
**Release note**:
```
release-note
```

### CODE REVIEW

File: cmd/sonobuoy/app/retrieve.go [hunk 1]
```diff
@@ -85,7 +85,7 @@ func retrieveResults(cmd *cobra.Command, args []string) {
 
 	err = eg.Wait()
 	if _, ok := err.(exec.CodeExitError); ok {
-		fmt.Fprintln(os.Stderr, "Results not ready yet. Check `sonobuoy status` for status.")
+		fmt.Fprintln(os.Stderr, "Results not ready yet at /tmp/sonobuoy/. Check `sonobuoy status` for status.")
```
Comment (timothysc) [inline cmd/sonobuoy/app/retrieve.go]: Well... that might be confusing, and it's customizable.

Comment (rajibmitra) [reply cmd/sonobuoy/app/retrieve.go]: Can I add parameters there ? the customized path you show there in stderr. what do you suggest ?

**Comment by chuckha** [comment]
@rajibmitra Thank you for your contribution!

The updated wording adds a bit of confusion because it implies `/tmp/sonobuoy` will exist on the local machine when we are actually looking at the sonobuoy container. Also, like @timothysc mentioned, the path is customizable so we really should be using the customized attribute in this message. You might consider a message like "Check $somepath on the sonobuoy container" where $somepath is the customized variable. Lastly, you will need to add a sign off to your commit before we can merge it. Please see [sign-off docs](https://github.com/heptio/sonobuoy/blob/master/CONTRIBUTING.md#dco-sign-off). 

If you make those changes I think this would be an improvement in the user experience as it gives them more information to use while debugging.

File: cmd/sonobuoy/app/retrieve.go (hunk 1 above)
Comment (timothysc) [reply cmd/sonobuoy/app/retrieve.go]: There is a config variable that is used to indicate output location.  

https://github.com/heptio/sonobuoy/blob/master/pkg/config/config.go#L276

**Comment by johnSchnake** [comment]
Closing due to age; welcome to reopen/resubmit if you think this tweak will clarify things.

**Pull request closed without merging**